CONTRIBUTING.md

```diff
@@ -72,7 +73,7 @@ able to control the mobile app.
 
 ## Full App Development
 
-### Pre-requisites
+### <span id="preReq">Pre-requisites</span>
```

Comment:
```suggestion
### Pre-requisites
```

Github auto-generates anchor links for headings. Spaces are replaced with `-` and anything that is not `a-z|A-Z|0-9` is removed. see https://gist.github.com/asabaylus/3071099#gistcomment-1593627